src/main/java/me/noodles/gui/PunishmentGUI.java

```diff
@@ -41,15 +38,15 @@ public void onEnable() {
         instance = this;
         MetricsLite metrics = new MetricsLite(this);
         Logger.log(Logger.LogLevel.INFO, "Managers Registered!");
+        Logger.log(Logger.LogLevel.INFO, "Loading Config's...");
+        this.createFiles();
+        Logger.log(Logger.LogLevel.INFO, "Config's Registered!");
```

Comment:
Moved config creation to the top as to not get any NPE's in methods trying to access configurations not yet created.